fbfmaproom/environment_linux.yml

```diff
@@ -181,19 +185,12 @@ dependencies:
   - xorg-xproto=7.0.31=h7f98852_1007
   - xz=5.2.5=h516909a_1
   - yaml=0.2.5=h7f98852_2
-  - zarr=2.11.1=pyhd8ed1ab_0
-  - zipp=3.7.0=pyhd8ed1ab_1
-  - zlib=1.2.11=h36c2ea0_1013
+  - zarr=2.11.3=pyhd8ed1ab_0
+  - zipp=3.8.0=pyhd8ed1ab_0
+  - zlib=1.2.11=h166bdaf_1014
   - zstd=1.5.2=ha95c52a_0
   - pip:
-    - dash-bootstrap-components==1.0.3
-    - dash-core-components==2.0.0
-    - dash-html-components==2.0.0
-    - dash-leaflet==0.1.23
-    - dash-table==5.0.0
-    - geobuf==1.1.1
-    - protobuf==3.19.4
     - pyaconf==0.7.1
     - pyyaml==6.0
     - queuepool==1.3.1
-prefix: /home/aaron/scratch/miniconda3/envs/fbfmaproom2
+prefix: /home/aaron/scratch/miniconda3/envs/dashtest
```

Comment:
@kgraaf commented that he usually deletes this line. My response is that it doesn't seem to make a difference, so I'd just as soon not bother with that manual step every time we update anything.